server/src/main/java/io/seata/server/session/SessionHolder.java

```diff
@@ -117,83 +118,122 @@ public static void init(String mode) throws IOException {
             // unknown store
             throw new IllegalArgumentException("unknown store mode:" + mode);
         }
-        reload();
+        reload(storeMode);
     }
 
+    //region reload
+
     /**
      * Reload.
      */
-    protected static void reload() {
+    protected static void reload(StoreMode storeMode) {
         if (ROOT_SESSION_MANAGER instanceof Reloadable) {
-            ((Reloadable)ROOT_SESSION_MANAGER).reload();
+            ((Reloadable) ROOT_SESSION_MANAGER).reload();
+        }
```

Comment:
So only in file mode do we need to reload, right?

Reply:
yes

Reply:
redis and db mode, only need to remove the global sessions in error state.